cookbook/configuration/web_server_configuration.rst

```diff
@@ -310,10 +316,10 @@ The **minimum configuration** to get your application running under Nginx is:
 .. tip::
 
     This executes **only** ``app.php``, ``app_dev.php`` and ``config.php`` in
-    the web directory. All other files will be served as text. You **must**
-    also make sure that if you *do* deploy ``app_dev.php`` or ``config.php``
-    that these files are secured and not available to any outside user (the
-    IP address checking code at the top of each file does this by default).
+    the web directory. All other files will be denied. You **must** also make 
+    sure that if you *do* deploy ``app_dev.php`` or ``config.php`` that these 
+    files are secured and not available to any outside user (the IP address 
+    checking code at the top of each file does this by default).
```

Comment:
Hmm, perhaps we should just talk about the end result to avoid.

1) Move this out of a `tip` block - just make it an explanation
2) Turn the following into a `caution` block.

```rst
.. caution::

    After you deploy to production, make sure that you **cannot** access the ``app_dev.php``
    or ``config.php`` scripts (i.e. ``http://example.com/app_dev.php`` and ``http://example.com/config.php``).
    If you *can* access these, be sure to remove the ``DEV`` section from the above configuration.
```